src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+
+  if (days >= 7) {
+    return days * dayPrice - 50;
+  }
+
+  if (days >= 3) {
+    return days * dayPrice - 20;
+  }
+
+  return days * dayPrice;
```

Comment:
The function name 'calculateRentalCost' is descriptive and good, but the magic numbers within the function need to be replaced with descriptive constants to avoid hardcoding and to make the code more readable and maintainable.